openedxstats/apps/sites/tests.py

```diff
@@ -375,3 +379,33 @@ def test_site_get_geographies_method_with_unicode(self):
         self.assertEqual(str(sitegeozone1), "https://www.κόσμε.co---Greece")
         self.assertEqual(str(geozone2), "\u00e9")
 
+
+class OTChartTestCase(TestCase):
+    def setUp(self):
+        user = User.objects.create_user('testuser', 'testuser@edx.com', 'password')
+        if user is None:
+            self.fail("Could not create testuser in setUp()")
+        self.client.login(username='testuser', password='password')
+
+    def test_nothing_imported_from_ajax_call(self):
+        # Ajax request
+        response = self.client.post('/sites/ot_chart/', context_type='application/json',
+                                    HTTP_X_REQUESTED_WITH='XMLHttpRequest')
```

Comment:
setUp is only 2 lines now and is only used twice in the file, and this client.post is only used twice too, does it make sense to abstract already?